angular.json

```diff
@@ -47,6 +47,11 @@
                 "glob": "**/*",
                 "input": "./shared/images/",
                 "output": "/assets/images/"
+              },
+              {
+                "glob": "**/*",
```

Comment:
Why to you need the above one then. ./shared/images are moved to /assets/images
`{
                 "glob": "**/*",
                 "input": "./shared/images/",
                 "output": "/assets/images/"
               },`

Reply:
In portal we are using image path as : assets/images/imageName and in player we are using assets/imageName so

Reply:
Let me fix this issue completely because in knowlg repository wherever we are using assets we are appending with **../assets/images/..** and in players we are using **../assets/.**. but as are exporting as **../assets/..** from angular.json in knowlg its breaking as part of players so , so let me  fix this issue i will share changes

Reply:
Now pushed changes , please review